packages/core/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -24,6 +24,7 @@ import { useNavigate } from 'react-router';
 interface IErrorPageProps {
   status: string;
   statusMessage: string;
+  docPath?: string;
```

Comment:
Maybe this should be renamed to something like `resourcePath` or `path` to not make it specific to docs if anybody else wants to use it.

Reply:
@hooloovooo I did think about calling it something more generic since it will just take any string and work with it. I liked `additionalInfo`.

Reply:
Yea that might also be an option. Not sure if we want it to be too generic though since that could easily make it inconsistent. Anything is probably fine for now and we can change it at a later point if we feel that it's too generic. If you change it to `additionalInfo` i'll approve and merge. :)

Reply:
@hooloovooo thank you for the assistance on this. Hopefully, it does the job for now!